src/api/credentialManager.ts

```diff
@@ -1,10 +1,10 @@
-import {Configuration} from "./generated/configuration"
+import { Configuration } from './generated/configuration';
```

Comment:
```suggestion
import { Configuration } from 'generated/configuration';
```

would this work?

Reply:
I don't think so, any path without the leading dot is treated like an external module (node_modules)

Reply:
I found some documentation if you are interested at https://www.typescriptlang.org/docs/handbook/module-resolution.html